concent_api/core/models.py

```diff
@@ -618,54 +667,6 @@ def clean(self) -> None:
             })
 
 
-class PaymentInfo(Model):
```

Comment:
Why have you moved the whole model?

Reply:
It is answered in https://github.com/golemfactory/concent/pull/1089#discussion_r251551319